### PR TITLE
update team members

### PR DESCRIPTION
**Pull request by EmelineAurore**
* team member list updated
* picture sizes modified for @bmpvieira and @estolle

### CODE REVIEW

**Comment by yeban** [comment]
I had a quick look. Looks good, except I guess Fabio's about should start with an uppercase.

I am merging this though. It's a minor issue, and I hope there will be a quick follow up.

**Comment by yeban** [comment]
Hmm. Broke the teams page.

File: team/index.html [hunk 1]
```diff
@@ -37,34 +37,17 @@
...
-- name: Julian Mazzitelli
-  img: julian-mazzitelli.jpg
-  about: 3rd year undergrad, Bioinformatics and Computational Biology Specialist, University of Toronto. Working with Wurm Lab for <a href="https://summerofcode.withgoogle.com/projects/#6585953724399616">Google Summer of Code Bionode project</a>. Creator of <a href="https://github.com/biorender/biorender">biorender</a>. [<a href="https://github.com/thejmazz">GitHub</a>] [<a href="https://twitter.com/thejmazz">Twitter</a>]
-
 - name: Karina Zile
   img: karina-zile.jpg
   about: <a href="http://lido-dtp.ac.uk/">LIDo</a> collaborative PhD student working on taxonomically restricted genes. Mathematician by training I am interested in answering biological questions using quantitative approaches. [<a href="https://uk.linkedin.com/in/karina-zile-a9116352">LinkedIn</a>]
 
+  - name: Fabio Manfredini
+  img: fabiomanfredini.jpg
```
Comment (IsmailM) [inline team/index.html]: missing leading white-space...

File: team/index.html [hunk 2]
```diff
@@ -37,34 +37,17 @@
...
+  about: broadly interested in the evolution of complex behaviors and phenotypic plasticity in social insects. In my studies I adopt a multidisciplinary perspective, that includes genomics and molecular, physiological and behavioral approaches.
```
Comment (IsmailM) [inline team/index.html]: missing leading white-space

**Comment by yannickwurm** [comment]
Please unmerge this asap guys

**Comment by yeban** [comment]
Reverted.

**Comment by yannickwurm** [comment]
thanks - how can we test this so that we can ensure that it works before pushing to public website?

**Comment by IsmailM** [comment]
Test locally first... (i.e. `bundle install; jekyll serve -w`, then go to `localhost:4000`)

Basically, it's yaml and the indentation is wrong (fixed in #42).

**Pull request merged**